lib/web/apiserver.go

```diff
@@ -2239,7 +2239,17 @@ func (h *Handler) deleteWebSession(w http.ResponseWriter, r *http.Request, _ htt
 
 func (h *Handler) logout(ctx context.Context, w http.ResponseWriter, sctx *SessionContext) error {
 	if err := sctx.Invalidate(ctx); err != nil {
-		return trace.Wrap(err)
+		h.log.
+			WithError(err).
+			WithField("user", sctx.GetUser()).
+			Warn("Failed to invalidate sessions")
```

Comment:
this is what the log looks like 
<img width="1377" alt="image" src="https://github.com/gravitational/teleport/assets/43280172/5a99f3be-1a77-45d7-a77d-601c7f9f80e6">